INSTALL

```diff
@@ -163,64 +225,64 @@ You can run the following command:
 sudo ldconfig ~/src/install-root/lib
 
 
-Building Subsurface under MacOSX (using Homebrew)
--------------------------------------------------
+Building Subsurface under MacOSX
+--------------------------------
 
-0) You need to have XCode installed. The first time (and possibly after updating OSX)
-you need to run
+Simple version:
 
-$ xcode-select --install
+brew install libzip cmake libusb pkg-config automake libtool hidapi curl libssh2
 
-1)  Install Homebrew
 
-$ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+0) You need to have XCode installed. The first time (and possibly after updating OSX)
 
-2) Install needed dependencies
+0.1) run "xcode-select --install"
 
-$ brew install asciidoc libzip cmake libusb pkg-config automake libtool hidapi curl libssh2
+1) You also need to install pkg-config
```

Comment:
I am convinced that our main recommendation should be to use home-brew. Because it works. When I have a new Mac, getting it to build subsurface from scratch, this is really the fastest and easiest way. 

You may have your reasons not to use it. But I would consider this a special case. It's good to somewhere document how to do it. But people should not be encouraged to do the manual install unless they know what they are doing. Plus, using home-brew, it's also much easier to stay up to date.

Reply:
I politely disagree, homebrew is known to really polute your mac and cause problems if you e.g. do not want to use the newest verdion of a library (I was told that is the reason why build.sh has a -builddeps option)

It is fair to write, that it can be done with homebrew, and giving the new developer the choise.

Reply:
This is an interesting argument. For the longest time I would have agreed with Robert. But at this point I'll admit that I have spent so much time to make sure that the -build-deps option works flawlessly, that I'm ok with giving this equal billing.
BTW, the reason for that option is NOT that you might want an older version of a library (though that's ALSO a potential problem), the real problem FOR ME (as someone building distribution binaries) is that Homebrew always compiles everything for your current macOS version. The developers quite explicitly and intentionally reject the idea that you might want to build for an older version of macOS. Their vision is that every individual user "simply" installs everything using Homebrew. Since that is not what I want our users to have to do and since I always want to support users back to the oldest version that Qt supports (which is just about to bump from 10.10 to 10.11, sadly), I cannot use Homebrew for my "production builds". Which is why I spend all that time ensuring that we can build all of our dependencies in the first place. And I think I'll add QtWebKit as an option there as well.

Reply:
Sorry I should have written newest SDK, which was what I meant.

Adding QtWebKit as an option is a real good thing. I am playing with adding an option "-prerequites" that installs the tools needed.